src/components/Content/Pursuance/views/DiscussView.js

```diff
@@ -75,6 +75,14 @@ class DiscussView extends Component {
                   }}} />
               </span>
             </div>
+            <div className="subtasks-ctn">
+              {subtaskGids.length > 0 && <h4>Subtasks</h4>}
+            <ul className="subtasks-list">
```

Comment:
@Moe-Shoman Indent here please, since this `<ul>` is contained within the `<div>`.

Reply:
@elimisteve Ah, I thought the linter would take care of that